.github/workflows/tests.yml

```diff
@@ -0,0 +1,52 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - master
+      - '*.x'
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  tests:
+    runs-on: ubuntu-22.04
+
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [7.4, '8.0', 8.1, 8.2]
+
+    name: PHP ${{ matrix.php }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: dom, curl, libxml, mbstring, redis, zip
+          ini-values: error_reporting=E_ALL
+          tools: composer:v2
+          coverage: none
```

Comment:
PHPUnit is called with options that generate a coverage report. Let's install Xdebug or pcov?

Reply:
Good catch!